apps/server/src/modules/user-import/config/user-import-config.ts

```diff
@@ -5,13 +5,13 @@ export const UserImportFeatures = Symbol('UserImportFeatures');
 export interface IUserImportFeatures {
 	userMigrationEnabled: boolean;
 	userMigrationSystemId: string;
-	instance: string;
+	useWithUserLoginMigration: boolean;
 }
 
 export class UserImportConfiguration {
 	static userImportFeatures: IUserImportFeatures = {
 		userMigrationEnabled: Configuration.get('FEATURE_USER_MIGRATION_ENABLED') as boolean,
 		userMigrationSystemId: Configuration.get('FEATURE_USER_MIGRATION_SYSTEM_ID') as string,
-		instance: Configuration.get('SC_THEME') as string,
+		useWithUserLoginMigration: Configuration.get('FEATURE_MIGRATION_WIZARD_WITH_USER_LOGIN_MIGRATION') as boolean,
```

Comment:
Can you put these configuration to the refactoring story ?

Reply:
If you mean the refactoring for the configs, then IUserImportFeatures is already mentioned